docker-compose.yml

```diff
@@ -28,7 +28,6 @@ services:
       - NODE_ENV=production
       - DATA_DIR=${DATA_DIR:-}
       - NETWORK=${NETWORK}
-      - RPC_ADDRESS=${RPC_ADDRESS:-}
```

Comment:
you should also remove it from the .env file samples as it won't be useful anymore